### PR TITLE
Brazilian portuguese translations

### PR DESCRIPTION
**Pull request by sauloachkar**
Some more fixes and new lines

### CODE REVIEW

File: src/locales/br/base_cards.json [hunk 1]
```diff
@@ -253,7 +253,7 @@
     "Increase your M€ production 2 steps.": "Aumente sua produção de M€ em 2 níveis.",
 
     "Electro Catapult": "Eletrocatapulta",
-    "Action: Spend 1 plant or 1 steel to gain 7 M€.": "Ação: gaste 1 planta ou 1 de aço para ganhar 7 M€.",
+    "Action: Spend 1 plant or 1 steel to gain 7M€.": "Ação: gaste 1 planta ou 1 de aço para ganhar 7 M€.",
```
Comment (bafolts) [inline src/locales/br/base_cards.json:256]: The english key has a space between `7 M€`. That has to be included here for this translation to be mapped.

File: src/locales/br/colonies_corp.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 
 {
    "Polyphemos": "Polyphemos",
-   "You start with 50 M€. Increase your M€ production 5 steps. Gain 5 titanium.": "Você começa com 50 M€. Aumente sua produção de M€ em 5 níveis. Ganhe 5 de titânio.",
+   "You start with 50M€. Increase your M€ production 5 steps. Gain 5 titanium.": "Você começa com 50 M€. Aumente sua produção de M€ em 5 níveis. Ganhe 5 de titânio.",
```
Comment (bafolts) [inline src/locales/br/colonies_corp.json:4]: This space between `50` and `M€` needs to remain in place for this translation to be mapped.

File: src/locales/br/log_messages.json [hunk 3]
```diff
@@ -167,12 +166,27 @@
     "${0} gained ${1} TR": "${0} ganhou ${1} IT",
     "1 card(s) were discarded": "1 carta foi descartada",
 
-    "Spend 1 plant to gain 7 M€": "Gastar 1 planta para ganhar 7 megacréditos",
+    "Spend 1 plant to gain 7 mega credit": "Gastar 1 planta para ganhar 7 megacréditos",
```
Comment (bafolts) [inline src/locales/br/log_messages.json:169]: This key needs to remain `Spend 1 plant to gain 7 M€`. There was a recent merge to the `main` branch which made this key change.

File: src/locales/br/log_messages.json [hunk 4]
```diff
@@ -167,12 +166,27 @@
...
     "Spend plant": "Gastar planta",
-    "Spend 1 steel to gain 7 M€": "Gastar 1 aço para ganhar 7 megacréditos",
+    "Spend 1 steel to gain 7 mega credit": "Gastar 1 aço para ganhar 7 megacréditos",
```
Comment (bafolts) [inline src/locales/br/log_messages.json:171]: This key needs to remain `Spend 1 steel to gain 7 M€`. There was a recent merge to the `main` branch which made this key change.

**Review by bafolts — CHANGES_REQUESTED** [review]
This changes some of the english keys which came in through recent commits to the `main` branch. I would rebase this branch to the `main` branch. The english keys can't be modified or the translation will not map.

**Comment by sauloachkar** [comment]
> This changes some of the english keys which came in through recent commits to the `main` branch. I would rebase this branch to the `main` branch. The english keys can't be modified or the translation will not map.

Sorry about that. Did a find and replace on all the [number+M€] and added a space between on all the brazilian portuguese files.

**Review by bafolts — APPROVED** [review]

**Pull request merged**